trace_decoder/src/decoding.rs

```diff
@@ -305,7 +294,7 @@ impl ProcessedBlockTrace {
     fn update_txn_and_receipt_tries(
         trie_state: &mut PartialTrieState,
         meta: &TxnMetaState,
-        txn_idx: TxnIdx,
+        txn_idx: usize,
```

Comment:
It's my preference.

I don't think they're particularly useful when used like this - I prefer actually naming variables.
The above line is actually _more_ confusing with the type alias - is there a special `TxnIdx` struct?
No, it's just a `#define usize`, for no particular reason...

@BGluth let's have this discussion in one place

Reply:
Yeah fair enough. You're taking over the refactor, so you can have the creative license. :slightly_smiling_face:

While I agree with you about preferring indicating the usage of a variable on the variable name (since it means an extra indirection to read the type), there are a few contexts where we can't or it's hard to specify nicely (eg. is the enum tuple variant expect a hashed slot or just a slot, or is `storage_accesses: Vec<(H256, StorageAccess)>,` using hashed or unhashed slots for keys?).

Reply:
Those make sense, I particularly think that e.g having `fn(&CodeHash) -> Vec<u8>` gets the point across. I might re-introduce them, or create newtypes as makes sense :)